fast_response/FastResponseAnalysis.py

```diff
@@ -992,6 +1020,424 @@ def unblind_TS(self, custom_events=None):
+            plt.sca(ax)
+            hp.visufunc.gnomview(map=outputProbHPMapPlotting,hold=True,title="Posterior Skymap",rot=(ramax*180/np.pi,decmax*180/np.pi,0),
+                xsize=nPixZoom,ysize=nPixZoom,reso=reso,bgcolor="white",badcolor="grey",margins=(.02,.02,.05,0),notext=True,cbar=False)
+            hp.graticule(verbose=True)
+            plotting_utils.plot_labels(decmax,ramax, reso,nPix=nPixZoom,label_scale=.97)
+            ax=plt.gca()
+            text_objs = ax.findobj(
+                lambda artist: isinstance(artist, Text)
+            )
+            for obj in text_objs:
+                obj.set_fontsize(30)
+            plotting_utils.plot_color_bar(range=[0,np.nanmax(outputProbHPMapPlotting)], cmap="viridis", col_label=r"Posterior PDF",
+                    offset=-40,labels=[0,"{0:.1e}".format(np.nanmax(outputProbHPMapPlotting)/2),"{0:.2e}".format(np.nanmax(outputProbHPMapPlotting))],loc=[0.86, 0.2, 0.03, 0.6])
+            theta, phi =plotting_utils.plot_contours([.9],outputProbHPMap)
+            hp.projplot(theta[0], phi[0], linewidth=2., c='k')
+            for i in range(1, len(theta)):
+                hp.projplot(theta[i], phi[i], linewidth=2., c='k', label=None)            
+            events = self.llh.exp
+            events = events[(events['time'] < self.stop) & (events['time'] > self.start)]
+
+            plotting_utils.plot_events2(events['dec'], events['ra'], events['sigma']*self._angScale, 
+                                        ramax,decmax, 2*6, sigma_scale=1.0,
+                    constant_sigma=False, same_marker=True, energy_size=True, col = ["red"]*len(events["dec"]),
+            resolution=reso
+            )
+            plotting_utils.plot_events2([np.radians(-36.61)],[np.radians(243.28)], [np.radians(.3)], 
+                                        ramax,decmax, 2*6, sigma_scale=1.0,
+                    constant_sigma=False, same_marker=True, energy_size=True, col = ["yellow"]) #plot the cascade
+            plt.savefig(plotting_location+"PosteriorMapZoomedWithEvents.png")
+            plt.close()
+
+
+            
+
+
+
+            fig, ax=plt.subplots(1,1,figsize=(10,10))
+            plt.sca(ax)
+            hp.visufunc.gnomview(map=np.log10(outputProbHPMapPlotting),hold=True,title="Posterior Skymap (Log p)",rot=(ramax*180/np.pi,decmax*180/np.pi,0),
+                xsize=nPixZoom,ysize=nPixZoom,reso=reso,bgcolor="white",badcolor="grey",margins=(.02,.02,.05,0),notext=True,cbar=False)
+            hp.graticule(verbose=True)
+            plotting_utils.plot_labels(decmax,ramax, reso,nPix=nPixZoom,label_scale=.97)
+            ax=plt.gca()
+            text_objs = ax.findobj(
+                lambda artist: isinstance(artist, Text)
+            )
+            for obj in text_objs:
+                obj.set_fontsize(30)
+            plotting_utils.plot_color_bar(range=[0,np.nanmax(np.log10(outputProbHPMapPlotting))], cmap="viridis", col_label=r"Posterior PDF",
```

Comment:
this function wraps around longer than a line. Can you separate some of the args onto their own lines?